mappings/net/minecraft/entity/ItemEntity.mapping

```diff
@@ -18,6 +20,13 @@ CLASS net/minecraft/class_1893 net/minecraft/entity/ItemEntity
 	METHOD method_7759 setPickupDelay (I)V
 		ARG 1 pickupDelay
 	METHOD method_7761 setItemStack (Lnet/minecraft/class_2056;)V
+	METHOD method_7762 setOwner (Ljava/lang/String;)V
+		ARG 1 owner
+	METHOD method_7763 setThrower (Ljava/lang/String;)V
```

Comment:
this doesnt feel like a yarn name. maybe Original owner? ask @shedaniel about this

Reply:
from nbt